src/interfaces/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from './temperature-reading'
+export * from './temperature-summary'
```

Comment:
We need to be careful when using barrel files, we can get circular dependency problems and in some bundlers, we can lose bundling optimizations like in Vite.

This article points to some issues, https://tkdodo.eu/blog/please-stop-using-barrel-files.
I wouldn't say we should not use barrel files, just to take care how we are using them.

Reply:
Taking a look, Thanks !